webvtt/reader.go

```diff
@@ -93,62 +83,86 @@ func (r *reader) parse(lines []string) ([]*caps.Caption, error) {
 	return captions, nil
 }
 
-func (r *reader) validateTimings(caption *caps.Caption, lastStartTime float64) error {
-	// FIXME: we might need to use a *float64 for Start/End so we can check for unset values.
-	if caption.Start == nil {
-		return fmt.Errorf("invalid cue start timestamp")
+func (r *reader) Detect(content []byte) bool {
+	return strings.Contains(string(content), "WEBVTT")
```

Comment:
Can use `bytes.HasPrefix(content, "WEBVTT")` so that you don't need to cast a string